core/classes/Adapter.js

```diff
@@ -153,6 +173,11 @@ module.exports = class Adapter {
     }
     const fixedData = this._sanitize(identity, data)
 
+    const validation = this._validate(_.omit(this.schemas[identity], "required"), _.omitBy(fixedData, _.isNil))
+    if (!validation.passed) {
+      throw new Error(`Property ${validation.errors[0].dataPath} validation fail: ${validation.errors[0].message}`)
```

Comment:
Please ensure this message works well for the case of missing required property.

Reply:
Messages work fine